.agent/workflows/update-execute.md

```
@@ -3,15 +3,21 @@ description: Execution phase of /update — read update-plan.md, execute sync st
 ---
 
 > **Sub-workflow of `/update`** — not invoked directly. Reads `Scope` and follows Stage Resumption Protocol.
+> See also: [Status Lifecycle](./_shared/status-lifecycle.md) · [Operating Model](./_shared/workflow-operating-model.md)
 > **Estimated context: ~1.5K tokens**
 
 ---
 
 ## 0. Pre-Execution Validation
 
 ### Staleness Guard
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)